apiserver/pkg/storage/calico/globalNetworkPolicy_storage.go

```diff
@@ -90,8 +90,20 @@ func (gc GlobalNetworkPolicyConverter) convertToAAPI(libcalicoObject resourceObj
 	lcgGlobalNetworkPolicy := libcalicoObject.(*api.GlobalNetworkPolicy)
 	aapiGlobalNetworkPolicy := aapiObj.(*aapi.GlobalNetworkPolicy)
 	aapiGlobalNetworkPolicy.Spec = lcgGlobalNetworkPolicy.Spec
+	// Tier field maybe left blank when policy created via OSS libcalico.
```

Comment:
This comment is replicated in a few places and also doesn't really make sense in this context. Can probably just be "default the tier field if not specified"

Reply:
Fixed.